ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -3506,6 +3506,27 @@ tests:
       TAG_VERSION: '@4.15'
     test:
     - chain: openshift-e2e-test-qe-ui
+- as: vsphere-ipi-ovn-dualstack-privmaryv6-f28
+  cron: 39 16 6 * *
+  steps:
+    cluster_profile: vsphere
+    env:
+      E2E_RUN_TAGS: '@vsphere-ipi and @network-ovnkubernetes and not @fips'
+      IP_FAMILIES: DualStackIPv6Primary
+      TAG_VERSION: '@4.15'
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack
+- as: vsphere-ipi-ovn-dualstack-privmaryv6-f28-destructive
+  cron: 18 17 5 * *
```

Comment:
On my local, the generated cron is different with yours:
```
$ generate-cron-entry.sh vsphere-ipi-ovn-dualstack-privmaryv6-f28-destructive openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml
cron: 26 19 28 * *
```
Is there something missing?

Reply:
```
vsphere-ipi-ovn-dualstack-privmaryv6-f28 ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml       cron: 12 22 9 * *
vsphere-ipi-ovn-dualstack-privmaryv6-f28 ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml       cron: 32 0 30 * *
vsphere-ipi-ovn-dualstack-privmaryv6-f28-destructive ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml       cron: 39 16 6 * *
vsphere-ipi-ovn-dualstack-privmaryv6-f28-destructive ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml       cron: 18 17 5 * *
```

Reply:
```
vsphere-ipi-ovn-dualstack-privmaryv6-f28-destructive ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml
TEST_NAME: vsphere-ipi-ovn-dualstack-privmaryv6-f28-destructive
YAML_FILE: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml
FN: 28
NUMBERS: 78413489286252525  -
MINUTE: 18
HOUR: 17
DAY_OF_MONTH: 5
MONTH: 6
DAY_OF_WEEK: 2

cron: 18 17 5 * *
```

Reply:
okay, when running `generate-cron-entry.sh`, the yaml file should use its basename. E.g:

```
$ generate-cron-entry.sh vsphere-ipi-ovn-dualstack-privmaryv6-f28-destructive openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml
cron: 26 19 28 * *
```

Not including `ci-operator/config/openshift/openshift-tests-private/` path

Reply:
updated